src/kriscv/kprovex/_default.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Final
+
+    from pyk.kast import KInner
+    from pyk.proof.reachability import APRProof
+
+    from .api import Config, Init, Show
+
+
+def init_from_claims(config: Config, spec_file: Path, claim_id: str) -> APRProof:
```

Comment:
All plugins implicitly support loading claims from K files.

Reply:
I notice that everywhere we have `claim_id` as a `str`, but not a `str | None`. Will this enable the case where a user wants to either (i) run the only claimthat is in th efile, or (ii) run all th eclaims in a given file?